client/javascripts/components/discussions/discussions.tag

```diff
@@ -15,10 +15,10 @@
         { comment.content }
       </div>
     </div>
-    <span if={ !opts.connected }>
+    <span if={ !connected() } class="discussion-commment__connection-needed">
```

Comment:
BEM stands for "Block, element and modifier". In this case,

- Block: discussion
- Element: comment
- Modifier: connection-needed

So it should look something like ```discussion__comment--connection-required```.